koku/masu/database/trino_sql/reporting_ocpazurecostlineitem_daily_summary.sql

```diff
@@ -599,6 +600,113 @@ JOIN cte_rankings as r
+    'Storage' as data_source,
+    'Storage unattributed' as namespace,
+    cast(NULL as varchar) as node,
+    cast(NULL as varchar) as persistentvolumeclaim,
+    cast(NULL as varchar) as persistentvolume,
+    cast(NULL as varchar) as storageclass,
+    cast(map(array[], array[]) as json) as pod_labels,
+    split_part(coalesce(nullif(azure.resourceid, ''), azure.instanceid), '/', 9) as resource_id,
+    date(coalesce(azure.date, azure.usagedatetime)) as usage_start,
+    date(coalesce(azure.date, azure.usagedatetime)) as usage_end,
+    {{bill_id | sqlsafe}} as cost_entry_bill_id,
+    coalesce(nullif(azure.subscriptionid, ''), azure.subscriptionguid) as subscription_guid,
+    max(azure.subscriptionname) as subscription_name,
+    max(json_extract_scalar(json_parse(azure.additionalinfo), '$.ServiceType')) as instance_type,
+    coalesce(nullif(azure.servicename, ''), azure.metercategory) as service_name,
+    azure.resourcelocation as resource_location,
+    sum(cast(coalesce(nullif(azure.quantity, 0), azure.usagequantity) as decimal(24, 9))) as usage_quantity,
+    max(CASE
+        WHEN split_part(azure.unitofmeasure, ' ', 2) = 'Hours'
+            THEN  'Hrs'
+        WHEN split_part(azure.unitofmeasure, ' ', 2) = 'GB/Month'
+            THEN  'GB-Mo'
+        WHEN split_part(azure.unitofmeasure, ' ', 2) != ''
+            THEN  split_part(azure.unitofmeasure, ' ', 2)
+        ELSE azure.unitofmeasure
+    END) as unit_of_measure,
+    coalesce(nullif(azure.billingcurrencycode, ''), nullif(azure.currency, ''), azure.billingcurrency) as currency,
+    sum(cast(coalesce(nullif(azure.costinbillingcurrency, 0), azure.pretaxcost) AS decimal(24,9)))  as pretax_cost,
+    sum(coalesce(nullif(azure.costinbillingcurrency, 0), azure.pretaxcost)) * cast({{markup}} as decimal(24,9)) as markup_cost,
+    sum(coalesce(nullif(azure.costinbillingcurrency, 0), azure.pretaxcost)) * cast({{markup}} as decimal(24,9)) as project_markup_cost,
+    sum(coalesce(nullif(azure.costinbillingcurrency, 0), azure.pretaxcost)) as pod_cost,
+    json_parse(azure.tags),
+    cast(NULL as integer) as cost_category_id,
+    cast({{azure_source_uuid}} as UUID)
+FROM hive.{{schema | sqlsafe}}.azure_line_items as azure
+LEFT JOIN cte_filtered_data as filtered
+    ON date(coalesce(azure.date, azure.usagedatetime)) = filtered.usage_date
+    AND coalesce(nullif(azure.resourceid, ''), azure.instanceid) = filtered.resource_id
+WHERE azure.source = {{azure_source_uuid}}
+    AND azure.year = {{year}}
+    AND azure.month = {{month}}
+    AND coalesce(azure.date, azure.usagedatetime) >= TIMESTAMP '{{start_date | sqlsafe}}'
+    AND coalesce(azure.date, azure.usagedatetime) < date_add('day', 1, TIMESTAMP '{{end_date | sqlsafe}}')
+    AND coalesce(nullif(azure.servicename, ''), azure.metercategory) LIKE '%Storage%'
+    AND coalesce(nullif(azure.resourceid, ''), azure.instanceid) LIKE '%%Microsoft.Compute/disks/%%'
+    AND filtered.usage_date IS NULL
```

Comment:
Maybe I don't understand exactly what data were looking for, but isn't this just Azure storage data for a virtual machine that had no relationship to the OpenShift clusters we have information for? Does it make sense to show cost information from resources that could be completely unrelated to OpenShift?

Reply:
From my understand the point of unattributed cost is track the cloud node attached storage that can not be associated with an Openshift cluster.

Reply:
@myersCody what does `cloud node attached storage` mean?

Reply:
The node-attached storage in a cluster typically refers to the disks that are mounted to VMs serving as nodes within a cluster. Each cloud has their own tooling for creating these disks, for azure this is Managed Disks. I took unattributed "node attached" storage to mean what portion of the Managed disks cost reported in the Azure bill is not not correlated with an OpenShift cluster.

Reply:
Recap:
This review comment prompted a lot of great discussions that lead down a path of us updating and refining how we define Unattributed node based storage. These details have been captured and added to the design doc.